src/pcapi/sandboxes/scripts/utils/storage_utils.py

```diff
@@ -1,27 +1,23 @@
 import os
+from pathlib import Path
 
 from pcapi.connectors.thumb_storage import create_thumb
 from pcapi.core.object_storage import store_public_object
 from pcapi.utils.human_ids import humanize
 from pcapi.utils.string_processing import get_model_plural_name
 
 
-def store_public_object_from_sandbox_assets(folder, model, offer_type):
+def store_public_object_from_sandbox_assets(folder, model, subcategoryId):
     mimes_by_folder = {"spreadsheets": "application/CSV", "thumbs": "image/jpeg", "zips": "application/zip"}
     plural_model_name = get_model_plural_name(model)
     thumb_id = humanize(model.id)
-    thumb_path = f"{os.path.dirname(os.path.realpath(__file__))}/../../{folder}/{plural_model_name}/{str(offer_type)}"
-
-    with open(thumb_path, mode="rb") as thumb_file:
+    thumbs_folder_path = Path(os.path.dirname(os.path.realpath(__file__))) / ".." / ".." / "thumbs"
```

Comment:
idem plus haut, ça me semblerait plus lisible et stable comme ça :

```suggestion
    import pcapi.sandboxes
    thumbs_folder_path = Path(pcapi.sandboxes.__path__[0]) / "thumbs"
```